node/prisma/README.md

```diff
@@ -0,0 +1,25 @@
+# Prerequisites
+1. Node/Npm
+2. YugabyteDB 
+
+# Build and run
+
+Install depedencies by running :
+```
+$ cd node/prisma && npm install
+```
+
+Specify the `DATABASE_URL` in the env file by the following command: 
+```
+echo 'DATABASE_URL="postgresql://<user>:<password>@<host>:<port>/<db_name>"' > .env 
```

Comment:
You can provide the exact string here so that users do not have to think what value to put here. Also, I think we started with a example-specific database name, so you can put `ysql_prisma` here. In that case, need to ask reader to explicitly create it beforehand.